src/Controller/SentencesController.php

```diff
@@ -373,13 +373,13 @@ public function edit_sentence($type = 'html')
      *
      * @return void
      */
-    public function adopt($id)
+    public function adopt($id, $type = 'html')
```

Comment:
There are better ways transmit the information "I want json" or "I want html". Although I never played with that, it’s all [explained in the docs](https://book.cakephp.org/3/en/views/json-and-xml-views.html). Basically we have the HTML header `Accept` for that already, or we can use CakePHP’s Routing File Extensions and append `.json` to the requested URL.

From what I’ve seen, the old design uses `Accept: */*` while the new design uses `Accept: application/json, text/plain, */*`, so you could already use that to decide what to return.